rosdep/python.yaml

```
@@ -7773,6 +7773,13 @@ python3-sklearn:
     pip:
       packages: [scikit-learn]
   ubuntu: [python3-sklearn]
+python3-smbus:
+  arch: [i2c-tools]
+  debian: [python3-smbus]
+  fedora: [python3-i2c-tools]
+  gentoo: [i2c-tools]
+  opensuse: [i2c-tools]
```

> It appears that only Debian and Ubuntu package this separately, other distros seem to have the Python bindings as part of the i2c-tools package, so I depended on that.

This isn't the case for openSUSE either: https://software.opensuse.org/package/python3-smbus
```suggestion
  opensuse: [python3-smbus]
```

I don't really know of a good way to check Gentoo, so the only platform that seems to provide the module in `i2c-tools` directly is Arch. If there's a decent chance the rule is not correct, I'd rather omit the Gentoo rule until someone from the community can show us how to verify it.